aws/main.tf

```diff
@@ -1,3 +1,23 @@
+
+terraform {
+  required_version = ">= 0.13.1, <= 2.0.0"
+
+  required_providers {
+    aws    = ">= 3.22.0, <5.0.0"
+    random = "~> 3.0"
+    http   = "~> 2.1"
+  }
```

Comment:
it seems we miss some versions in `versions.tf` are they no longer required?

Reply:
No longer required indeed